tests/Unit/Contrib/Zipkin/ZipkinSpanConverterTest.php

```diff
@@ -61,14 +61,14 @@ public function test_should_convert_a_span_to_a_payload_for_zipkin(): void
             ->addEvent('validators.list', Attributes::create(['job' => 'stage.updateTime']), 1505855799433901068)
             ->setHasEnded(true);
 
-        $converter = new SpanConverter('test.name');
+        $converter = new SpanConverter();
         $row = $converter->convert([$span])[0];
 
         $this->assertSame($span->getContext()->getSpanId(), $row['id']);
         $this->assertSame($span->getContext()->getTraceId(), $row['traceId']);
         $this->assertSame('1000000000000000', $row['parentId']);
 
-        $this->assertSame('test.name', $row['localEndpoint']['serviceName']);
+        $this->assertSame('unknown_service', $row['localEndpoint']['serviceName']);
```

Comment:
you might need to merge in main branch, this is something I changed in a commit in the last day or so... `unknown_service:php`

Reply:
yes, that's right. Will do that when this PR will be approved